BoxContentSDK/BoxContentSDK/Models/BOXRepresentation.h

```diff
@@ -9,7 +9,7 @@
 #import <Foundation/Foundation.h>
 #import "BOXContentSDKConstants.h"
 
-@interface BOXRepresentation : NSObject
+@interface BOXRepresentation : NSObject <NSCoding>
```

Comment:
Could we use `NSSecureCoding`? It only really requires using `decodeObjectOfClass:forKey:` below and implementing `+ (BOOL)supportsSecureCoding { return YES; }`.